aws/resource_aws_ebs_volume.go

```diff
@@ -44,6 +44,9 @@ func resourceAwsEbsVolume() *schema.Resource {
 				Type:     schema.TypeInt,
 				Optional: true,
 				Computed: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
This is to address the comment below pertaining to `iops` values for non Io1 volume types, correct?

Reply:
This allows operators to continue using syntax like the below in Terraform 0.11 with modules 👍 

```
variable "iops" {
  default = "0"
}

variable "type" {
  default = "standard"
}

resource "aws_ebs_volume" "example" {
  # ... other configuration ...
  iops = "${var.iops}"
  type = "${var.type}"
}
```

```
module "example1" {
  # ...
  iops = "1000"
  type = "io1"
}

module "example2" {
  # ...
  type = "gp2"
}
```

Selectively choosing when to save the information into the Terraform state was a workaround to suppressing the difference in certain cases, although the selectiveness itself can cause issues after updates (e.g. not refreshing attributes appropriately).